### DISCUSSION TITLE
How do I get the outline to dynamically update.

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#917] [Q&A]
I cannot work out how the outline itself.

When coding, new fields are constantly added but the outline is not being updated or its updated a while later. 
I find I need to close and open the file again to get it updated.

TIA, Darryl.

**Comment by worksofliam** [comment]
@dhfreinkel Is this a new issue or something you've been having before today?

Generally, the outline will update after you save, switch tabs or wait a short while.

**Comment by dhfreinkel** [comment]
It happens from time to time. Today I needed to change a few hundred. I had to exit and use another editor to do the replace.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562
